.travis.yml

```diff
@@ -1,35 +1,33 @@
-sudo: false
+sudo:
+    - false
 
-language: cpp
+language:
+  - cpp
 
 cache:
-  directories:
-    - odb
+    directories:
+        - odb
 
 matrix:
   include:
 
-    # Clang 3.9
-    - compiler: clang
-      env: C_COMPILER=clang-3.9 CXX_COMPILER=clang++-3.9
-      addons:
-        apt:
-          sources:
-            - boost-latest
-            - ubuntu-toolchain-r-test
-            - llvm-toolchain-trusty-3.9
-          packages:
-            - clang-3.9
-            - libboost1.55-dev
-            - libboost-filesystem1.55-dev
-            - libboost-iostreams1.55-dev
-            - libboost-program-options1.55-dev
-            - libboost-regex1.55-dev
-            - libboost-log1.55-dev
-            - ninja-build
+      # OS X
```

Comment:
Warning during build:

> CMake Warning at /usr/local/Cellar/cmake/3.10.2/share/cmake/Modules/FindBoost.cmake:801 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.2/share/cmake/Modules/FindBoost.cmake:907 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.10.2/share/cmake/Modules/FindBoost.cmake:1542 (_Boost_MISSING_DEPENDENCIES)
  CMakeLists.txt:50 (find_package)

Maybe you also need to list which version of Boost has to be installed?